src/radical/entk/utils/profiler.py

```diff
@@ -36,25 +36,26 @@ def _initialize(self):
     def _get_resource_manager_details(self):
     
         self._rman_df = pd.read_csv('%s/radical.entk.resource_manager.0000.prof'%self._src,skiprows=[0,1], 
-                            names=['timestamp', 'thread/proc', 'uid', 'state', 'event', 'msg']
+                            names = ['timestamp', 'event', 'comp', 'tid', 'uid', 'state', 'msg']
+                            # names=['timestamp', 'thread/proc', 'uid', 'state', 'event', 'msg']
```

Comment:
This file is interesting - it looks like a mix of EnTK specifika, RU profiler and RA.  I am not sure I understand the purpose, and would think that some parts should be moved to RU and RA, respectively and that only the EnTK specific parts should remain here?